ci-operator/step-registry/upi/deprovision/aws/stage1/upi-deprovision-aws-stage1-ref.yaml

```diff
@@ -0,0 +1,12 @@
+ref:
+  as: upi-deprovision-aws-stage1
+  from: upi-installer
+  grace_period: 10m
+  commands: upi-deprovision-aws-stage1-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: >-
+    This step delets stacks before running `openshift-install destroy` command
+    e.g. *-security *-bootstrap, *-control-plane *-compute* *apps-dns stacks.
```

Comment:
instead of being opinionated about what you're likely to pick up, it might be easier to maintain if you doc the `${SHARED_DIR}/to_be_removed_cf_list_stage_1` input format, and then leave it to folks wiring up workflows to make the match between "step A creates `${SHARED_DIR}/${pathA}`, and step B consumes it".